server/data/accreditedProgrammesApi/courseClient.test.ts

```diff
@@ -327,7 +333,11 @@ pactWith({ consumer: 'Accredited Programmes UI', provider: 'Accredited Programme
   })
 
   describe('updateParticipation', () => {
-    const courseParticipation = courseParticipationFactory.build({ id: '0cff5da9-1e90-4ee2-a5cb-94dc49c4b004' })
+    const courseParticipation = courseParticipationFactory.build({
+      id: '0cff5da9-1e90-4ee2-a5cb-94dc49c4b004',
```

Comment:
Does this need to be `cc8eb19e-050a-4aa9-92e0-c654e5cfe281`?

Reply:
Yep, good spot!